apps/barista-design-system/src/pages/overview-page/components/component-preview-tile/component-preview-tile.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../../../../libs/barista-components/core/src/style/colors';
+@import '../../../../styles/global/variables';
+
+$tilesize: 290px;
+$tilefontsize: 28px;
+$tilefontsize-big: 42px;
+$gridgap: 20px;
+$imagesize: 156px;
+
+@mixin tile-theme($link-color, $hover-color, $active-color) {
+  color: $link-color;
+
+  :hover,
```

Comment:
It is applied onto the host which has the border for example

Reply:
yeah, true that, but the selector `:hover` does nothing, since with the mixin applied it's gonna be:
```
:host {
  :hover, <-- this does nothing
  &hover {
    ...
  }
}
```

doesn't really matter though, could just save some lines of code 😉